src/bidiMapper/domains/network/NetworkManager.ts

```diff
@@ -78,13 +85,16 @@ export class NetworkManager {
           request.handleRedirect(params);
           networkManager.#networkStorage.deleteRequest(params.requestId);
           networkManager.#networkStorage
-            .createRequest(params.requestId, request.redirectCount + 1)
+            .createRequest(params.requestId, {
```

Comment:
why do we sometimes create requests directly through storage and sometimes though the network manager?

Reply:
Fixed